scst-store/multicluster-setup.hbs.md

```diff
@@ -108,6 +153,66 @@ certificate and authentication token.
 The cluster now has a CA certificate named  `store-ca-cert` and authentication
 token named `store-auth-token` in the namespace `metadata-store-secrets`.
 
+### Copy AMR Cloud Event Handler edit token from the View cluster
```

Comment:
Inside an paragraph, it's spelled CloudEvent. Camel case single word.
https://github.com/cloudevents/spec